docs/administration/security/authentication.md

```diff
@@ -429,6 +429,10 @@ The [PAM](#pam) section is a useful comparison as it uses the same method to com
 
 ### Active Directory
 
+:::tip
+The maximum quantity of results that Rundeck is able to retrieve from Active Directory is limited to the value specified for the MaxPageSize parameter within the Active Directory configuration.
```

Comment:
This needs more details. Where is this setting inside Active Directory?  How would they change it?  Preferably point to Microsoft Docs that discuss this setting.